keepvariable/keepvariable_core.py

```diff
@@ -196,9 +195,8 @@ def parse_saved_value(self, value, additional_params: Optional[dict] = None):
 
         return value
 
-    def decode_loaded_value(
-        self, value: str
-    ) -> Union[dict, pd.DataFrame, np.ndarray, datetime.datetime]:
+    def decode_loaded_value(self,
+                            value: str) -> Union[dict, pd.DataFrame, np.ndarray, datetime.datetime]:
```

Comment:
previous formating seemed nicer (ending bracket on a new line

Reply:
Both are terrible

Reply:
but this new one is much better on other places